Learning/data_collection.py

```diff
@@ -54,8 +55,8 @@ def evaluate_policy(self, env, policy, collection_time_steps, sector_rows, secto
     # import os
     # cpu_cores =  [i for i in range(30, 61)]
     # os.system("taskset -pc {} {}".format(",".join(str(i) for i in cpu_cores), os.getpid()))
-    rows = 150
-    cols = 300
+    rows = 30
+    cols = 60
```

Comment:
Why 30 & 60?

Reply:
Sorry didn't change that again, I used it for:
- faster testing
- higher chance of plants sharing space